numfmt.go

```diff
@@ -4633,6 +4637,26 @@ var (
 	}
 )
 
+// getSupportedLanguageInfo returns language infomation by giving language code.
+func getSupportedLanguageInfo(lang string) (languageInfo, bool) {
```

Comment:
The language info of number format code maybe represents different calendar type, this function should be handle this kind of number format. E.g. 3010429